osf/migrations/0002_auto_20170329_1729.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9 on 2017-03-29 22:29
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import django.utils.timezone
+import osf.utils.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('osf', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='osfuser',
+            name='mailing_lists',
+        ),
+        migrations.AlterField(
+            model_name='abstractnode',
+            name='date_created',
+            field=osf.utils.fields.NonNaiveDateTimeField(auto_now_add=True),
+        ),
+        migrations.AlterField(
+            model_name='abstractnode',
+            name='date_modified',
+            field=osf.utils.fields.NonNaiveDateTimeField(auto_now=True, db_index=True, default=django.utils.timezone.now),
+            preserve_default=False,
+        ),
+        migrations.AlterField(
+            model_name='apioauth2application',
+            name='date_created',
+            field=osf.utils.fields.NonNaiveDateTimeField(auto_now_add=True),
+        ),
+        migrations.AlterField(
+            model_name='comment',
+            name='date_created',
+            field=osf.utils.fields.NonNaiveDateTimeField(auto_now_add=True),
+        ),
+        migrations.AlterField(
+            model_name='comment',
+            name='date_modified',
+            field=osf.utils.fields.NonNaiveDateTimeField(auto_now=True),
+        ),
+        migrations.AlterField(
+            model_name='draftregistration',
+            name='datetime_initiated',
+            field=osf.utils.fields.NonNaiveDateTimeField(auto_now_add=True),
+        ),
+        migrations.AlterField(
+            model_name='draftregistration',
+            name='datetime_updated',
+            field=osf.utils.fields.NonNaiveDateTimeField(auto_now=True),
+        ),
+        migrations.AlterField(
+            model_name='fileversion',
+            name='date_created',
+            field=osf.utils.fields.NonNaiveDateTimeField(auto_now_add=True),
+        ),
+        migrations.AlterField(
+            model_name='guid',
+            name='created',
+            field=osf.utils.fields.NonNaiveDateTimeField(auto_now_add=True, db_index=True),
+        ),
+        migrations.AlterField(
+            model_name='nodelog',
+            name='date',
+            field=osf.utils.fields.NonNaiveDateTimeField(auto_now_add=True, db_index=True, null=True),
```

Comment:
This actually shouldn't be an `auto_now.*` field; it's possible for logs to be retroactively created by webhooks, which sets a different `date`. Fixing